arlo-client/src/components/CreateAudit.tsx

```diff
@@ -55,6 +97,32 @@ const CreateAudit = ({ history }: RouteComponentProps<ICreateAuditParams>) => {
       >
         Create a New Audit
       </Button>
+      {elections.elections.length > 0 ? (
+        <>
+          <RadioGroup
+            label="Select an existing audit:"
+            selectedValue={electionId}
+            onChange={e => setElectionId(e.currentTarget.value)}
+          >
+            {elections.elections.map(e => (
+              <Radio key={e.id} value={e.id}>
+                {e.name || <i>Not named yet</i>} (Created&nbsp;
+                {moment(e.date).format('MM/DD/YYYY')})
```

Comment:
No need for `moment`, and we get better localization support:

```suggestion
                {new Intl.DateTimeFormat().format(e.date)})
```